src/order/conditionally_complete_lattice.lean

```diff
@@ -1042,142 +1045,3 @@ noncomputable instance with_top.with_bot.complete_linear_order {α : Type*}
   .. with_top.linear_order }
 
 end with_top_bot
-
-section subtype
-variables (s : set α)
-
-/-! ### Subtypes of conditionally complete linear orders
-
-In this section we give conditions on a subset of a conditionally complete linear order, to ensure
-that the subtype is itself conditionally complete.
-
-We check that an `ord_connected` set satisfies these conditions.
-
-TODO There are several possible variants; the `conditionally_complete_linear_order` could be changed
```

Comment:
why did this todo get removed?

Reply:
Because these variants make sense for `Ixy`, where one of `x`/`y` is `c` (or the original order is bounded). So, I think that the new TODO about intervals covers all relevant use cases of the old TODO.